src/lines.jl

```diff
@@ -10,280 +10,293 @@ module Lines
     end
 
-    """Line spacing code"""
+    """Line spacing code (Getter)"""
     function Spacing()::String
         return Utils.get_string(Lib.Lines_Get_Spacing())
     end
 
-    """Line spacing code"""
+    """Line spacing code (Getter)"""
     function Spacing(Value::String)
         Lib.Lines_Set_Spacing(Cstring(pointer(Value)))
     end
 
-    """(read-only) Total Number of customers served from this line section."""
+    """Total Number of customers served from this line section."""
     function TotalCust()::Int
         return Lib.Lines_Get_TotalCust()
     end
 
+    """Units for Line (Getter)"""
     function Units()::Int
         return Lib.Lines_Get_Units()
     end
 
+    """Units for Line (Setter)"""
     function Units(Value::Int)
         Lib.Lines_Set_Units(Value)
     end
 
-    """Zero Sequence reactance ohms per unit length."""
+    """Zero Sequence reactance ohms per unit length. (Getter)"""
     function X0()::Float64
         return Lib.Lines_Get_X0()
     end
 
-    """Zero Sequence reactance ohms per unit length."""
+    """Zero Sequence reactance ohms per unit length. (Setter)"""
     function X0(Value::Float64)
         Lib.Lines_Set_X0(Value)
     end
 
-    """Positive Sequence reactance, ohms per unit length."""
+    """Positive Sequence reactance, ohms per unit length. (Getter)"""
     function X1()::Float64
         return Lib.Lines_Get_X1()
     end
 
-    """Positive Sequence reactance, ohms per unit length."""
+    """Positive Sequence reactance, ohms per unit length. (Setter)"""
     function X1(Value::Float64)
         Lib.Lines_Set_X1(Value)
     end
 
-    """Earth return reactance value used to compute line impedances at power frequency"""
+    """Earth return reactance value used to compute line impedances at power frequency (Getter)"""
     function Xg()::Float64
         return Lib.Lines_Get_Xg()
     end
 
-    """Earth return reactance value used to compute line impedances at power frequency"""
+    """Earth return reactance value used to compute line impedances at power frequency (Setter)"""
     function Xg(Value::Float64)
         Lib.Lines_Set_Xg(Value)
     end
 
+    """Susceptance matrix, ohms per unit length. Array of doubles. (Getter)"""
     function XMatrix()::Vector{Float64}
+        # TODO: should this method return Matrix{ComplexF64} with just imaginary components?
         return Utils.get_float64_array(Lib.Lines_Get_Xmatrix)
     end
 
+    """Susceptance matrix, ohms per unit length. Array of doubles. (Setter)"""
     function XMatrix(Value::Vector{Float64})
+        # TODO: should this method return Matrix{ComplexF64} with just imaginary components?
         Value, ValuePtr, ValueCount = Utils.prepare_float64_array(Value)
         Lib.Lines_Set_Xmatrix(ValuePtr, ValueCount)
     end
 
-    """Yprimitive: Does Nothing at present on Put; Dangerous"""
-    function Yprim()::Array{ComplexF64, 2}
+    """Yprimitive: Does Nothing at present on Put; Dangerous (Getter)"""
```

Comment:
The getter isn't dangerous, is it? If it is dangerous, maybe say why. For example, if it's dangerous to write to the array returned, then say that.

For the setter, if it doesn't do anything, why is that dangerous?

Reply:
pinging @PMeira to see why the docstrings here said these were dangerous.

Reply:
> Yprimitive: Does Nothing at present on Put; Dangerous (Getter)

@tshort @kdheepak That's imported from the COM helpstring. Looking into the SVN history, which starts on 2012-02-25, the dangerous comment was already in place.

From the source code, it can either return the current `Yprim` or null. It doesn't check if `YprimInvalid` is false, so that might be the dangerous part.

(The setter is indeed empty, does nothing useful)